digital-deck-ui/src/main.js

```diff
@@ -1,7 +1,12 @@
-import { createApp } from 'vue';
-import App from './App.vue';
-import store from './store';
+import { createApp } from 'vue'
```

Comment:
Can you put the `;` after every line please? I know it's not required for javascript but it's good practice